src/include/simeng/arch/aarch64/operandContainer.hh

```diff
@@ -10,7 +10,7 @@ namespace arch {
 namespace aarch64 {
 
 /** The maximum number of source registers a non-SME instruction can have. */
-const uint8_t MAX_SOURCE_REGISTERS = 6;
+const uint8_t MAX_SOURCE_REGISTERS = 7;
```

Comment:
Need to be careful this doesn't overwrite the changes in #429

Reply:
This PR is based on #429 and so cannot be merged before it. Will keep an eye out for this though